media/starboard/bidirectional_fit_reuse_allocator.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Let's seek help from @dahlstrom-g to bring the history of the original files for bidirectional_fit_reuse_allocator* and decoder_buffer_allocator* over.

Reply:
Done.